<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Welcome</class>
 <widget class="QDialog" name="Welcome">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>30</y>
     <width>111</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>welcome</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>90</y>
     <width>54</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>本机ip</string>
   </property>
  </widget>
  <widget class="QLabel" name="self_ip">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>90</y>
     <width>141</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>90</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>作为主机</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="host_ip">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>140</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>183.172.130.219</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>140</y>
     <width>54</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>主机ip</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="host_port">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>210</y>
     <width>51</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>1235</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>210</y>
     <width>54</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>主机端口</string>
   </property>
  </widget>
  <widget class="QPushButton" name="refresh_self_ip">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>140</y>
     <width>101</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>重新搜索本机ip</string>
   </property>
  </widget>
  <widget class="QPushButton" name="startbtn">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>250</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>开始连接</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
